neo/Prompt/Commands/BuildNRun.py

```diff
@@ -67,53 +86,48 @@ def BuildAndRun(arguments, wallet, verbose=True, min_fee=DEFAULT_MIN_FEE, invoca
           from_addr=None, min_fee=DEFAULT_MIN_FEE, invocation_test_mode=True,
           debug_map=None, invoke_attrs=None, owners=None):
-    test = get_arg(arguments, 1)
-
-    if test is not None and test == 'test':
-
-        if wallet is not None:
-
-            f_args = arguments[2:]
-            i_args = arguments[7:]
-
-            script = GatherLoadedContractParams(f_args, contract_script)
-
-            tx, result, total_ops, engine = test_deploy_and_invoke(script, i_args, wallet, from_addr,
-                                                                   min_fee, invocation_test_mode, debug_map=debug_map,
-                                                                   invoke_attrs=invoke_attrs, owners=owners)
-            i_args.reverse()
-
-            return_type_results = []
-
-            try:
-                rtype = ContractParameterType.FromString(f_args[1])
-                for r in result:
-                    cp = ContractParameter.AsParameterType(rtype, r)
-                    return_type_results.append(cp.ToJson())
-            except Exception as e:
-                logger.error('Could not convert result to ContractParameter: %s ' % e)
-
-            if tx is not None and result is not None:
-                if verbose:
-                    print("\n-----------------------------------------------------------")
-                    print("Calling %s with arguments %s " % (path, [item for item in reversed(engine.invocation_args)]))
-                    print("Test deploy invoke successful")
-                    print("Used total of %s operations " % total_ops)
-                    print("Result %s " % return_type_results)
-                    print("Invoke TX gas cost: %s " % (tx.Gas.value / Fixed8.D))
-                    print("-------------------------------------------------------------\n")
-
-                return tx, result, total_ops, engine
-            else:
-                if verbose:
-                    print("Test invoke failed")
-                    print("tx is, results are %s %s " % (tx, result))
 
+    if wallet is not None:
+
+        f_args = arguments[1:]
+        i_args = arguments[6:]
+
+        script = GatherLoadedContractParams(f_args, contract_script)
+
+        tx, result, total_ops, engine = test_deploy_and_invoke(script, i_args, wallet, from_addr,
+                                                               min_fee, invocation_test_mode, debug_map=debug_map,
+                                                               invoke_attrs=invoke_attrs, owners=owners)
+        i_args.reverse()
+
+        return_type_results = []
+        try:
+            rtype = ContractParameterType.FromString(f_args[1])
+            for r in result:
+                cp = ContractParameter.AsParameterType(rtype, r)
+                return_type_results.append(cp.ToJson())
+        except Exception as e:
+            print('Could not convert result to ContractParameter: %s ' % e)
```

Comment:
To make it re-usable we should not print but throw an exception so a user can choose how to handle it. For now I think we could throw a `TypeError`.